packages/shared/src/components/InteractionCounter.tsx

```diff
@@ -42,26 +45,24 @@ export default function InteractionCounter({
     setShownValue(value);
   };
 
+  const childClassName =
+    'h-5 inline-block transition-[opacity,transform] ease-in-out duration-300 will-change-[opacity,transform]';
+
   return (
-    <span
-      className={classNames(
-        'relative overflow-hidden',
-        styles.interactionCounter,
-        className,
-      )}
-      {...props}
-    >
+    <span className={elementClassName} {...props}>
       <span
-        className={
-          animate ? '-translate-y-full opacity-0' : 'translate-y-0 opacity-100'
-        }
+        className={classNames(
+          childClassName,
+          animate ? '-translate-y-full opacity-0' : 'translate-y-0 opacity-100',
+        )}
       >
         {shownValue}
       </span>
       <span
-        className={`absolute left-0 top-0 ${
-          animate ? 'translate-y-0 opacity-100' : 'translate-y-full opacity-0'
-        }`}
+        className={classNames(
+          childClassName,
+          animate ? '-translate-y-full opacity-100' : 'translate-y-0 opacity-0',
```

Comment:
Is this change from `'translate-y-0 opacity-100' : 'translate-y-full opacity-0'` to `'-translate-y-full opacity-100' : 'translate-y-0 opacity-0'` intended?

Reply:
Yes. Before it was using absolute positioning and that doesn't work if we want to center the value element.

So I switched to a `flex-col` layout instead. The `span` elements are rendered below each other. Hence when we animate, we need to `translate-y` both of the spans 100% up. Does that make sense?

Reply:
Makes sense in idea. For as long as you've tested this on both v1 and not v1 where it is used we should be fine 👍